<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSMExplorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon-128.png</normaloff>:/images/icon-128.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="SlippyMapTreeView" name="tvwMarkers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SlippyMapWidget" name="slippyMap" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 10px;
	background-color: palette(base);
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="zoomInButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/zoomin.png</normaloff>:/icons/zoomin.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="zoomOutButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/zoomout.png</normaloff>:/icons/zoomout.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="currentLocationButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/arrowdown.png</normaloff>:/icons/arrowdown.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>482</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Default</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="OEDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>456</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Object</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabShapeEditor">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="selectedObjectName"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="selectedObjectDescription"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Attributes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="selectedObjectAttributes"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Appearance</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Border Color</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Border Size</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sidebarSaveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sidebarResetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblNoShapeSelected">
       <property name="styleSheet">
        <string notr="true">QLabel {
	padding-top: 40px;
	padding-bottom: 40px;
}</string>
       </property>
       <property name="text">
        <string>Select a shape...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionObject_NewMarker">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/marker-add.svg</normaloff>:/icons/toolbar/marker-add.svg</iconset>
   </property>
   <property name="text">
    <string>New marker...</string>
   </property>
  </action>
  <action name="actionObject_NewMarkerAtCurrent">
   <property name="text">
    <string>New marker at current location</string>
   </property>
  </action>
  <action name="actionObject_Properties">
   <property name="text">
    <string>Properties...</string>
   </property>
  </action>
  <action name="actionObject_Delete">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
  <action name="actionMarkerExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionMarkerImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionMarkerManage">
   <property name="text">
    <string>Manage...</string>
   </property>
  </action>
  <action name="actionViewSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sidebar</string>
   </property>
  </action>
  <action name="actionDebugOpenDirectionsFile">
   <property name="text">
    <string>Open Directions File...</string>
   </property>
  </action>
  <action name="actionViewClearRoute">
   <property name="text">
    <string>Clear Route</string>
   </property>
  </action>
  <action name="actionFileSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionDrawLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/path-add.svg</normaloff>:/icons/toolbar/path-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Line</string>
   </property>
  </action>
  <action name="actionDrawEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/ellipse-add-extent.svg</normaloff>:/icons/toolbar/ellipse-add-extent.svg</iconset>
   </property>
   <property name="text">
    <string>Add Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionDrawRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/rect-add.svg</normaloff>:/icons/toolbar/rect-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Rectangle</string>
   </property>
  </action>
  <action name="actionDrawPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/polygon-add.svg</normaloff>:/icons/toolbar/polygon-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Polygon</string>
   </property>
  </action>
  <action name="actionMapLayersConfigure">
   <property name="text">
    <string>Configure...</string>
   </property>
  </action>
  <action name="actionFileLayersConfigure">
   <property name="text">
    <string>Configure...</string>
   </property>
  </action>
  <action name="actionFile_Print">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionFileExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionMapGpsAddSource">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/gps-add.svg</normaloff>:/icons/toolbar/gps-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Source...</string>
   </property>
  </action>
  <action name="actionViewGpsLog">
   <property name="text">
    <string>GPS Log</string>
   </property>
  </action>
  <action name="actionToolsOSMImport">
   <property name="text">
    <string>OSM Import...</string>
   </property>
  </action>
  <action name="actionLayer_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/layer-add.svg</normaloff>:/icons/toolbar/layer-add.svg</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionLayer_Delete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionLayer_Rename">
   <property name="text">
    <string>Rename...</string>
   </property>
  </action>
  <action name="actionLayer_SortAscending">
   <property name="text">
    <string>Sort Ascending</string>
   </property>
  </action>
  <action name="actionLayer_SortDescending">
   <property name="text">
    <string>Sort Descending</string>
   </property>
  </action>
  <action name="actionLayer_TileLayers_HideAll">
   <property name="text">
    <string>Hide All</string>
   </property>
  </action>
  <action name="actionLayer_TileLayers_ShowAll">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionFile_NewWorkspace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/new.svg</normaloff>:/icons/toolbar/new.svg</iconset>
   </property>
   <property name="text">
    <string>New Workspace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFile_OpenWorkspace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/open.svg</normaloff>:/icons/toolbar/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Workspace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFile_Recent_ClearMenu">
   <property name="text">
    <string>Clear Menu</string>
   </property>
  </action>
  <action name="actionWeather_ShowWFOGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Weather Stations</string>
   </property>
  </action>
  <action name="actionFile_SaveWorkspace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/save.svg</normaloff>:/icons/toolbar/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save Workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFile_SaveWorkspaceAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/save-as.svg</normaloff>:/icons/toolbar/save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save Workspace As...</string>
   </property>
  </action>
  <action name="actionFile_CloseWorkspace">
   <property name="text">
    <string>Close Workspace</string>
   </property>
  </action>
  <action name="actionShow_Alerts">
   <property name="text">
    <string>Show Alerts</string>
   </property>
  </action>
  <action name="actionLayer_PreviousFrame">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rewind.png</normaloff>:/icons/rewind.png</iconset>
   </property>
   <property name="text">
    <string>Previous Frame</string>
   </property>
  </action>
  <action name="actionLayer_NextFrame">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/fforward.png</normaloff>:/icons/fforward.png</iconset>
   </property>
   <property name="text">
    <string>Next Frame</string>
   </property>
  </action>
  <action name="actionLayer_Play">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionImport_GPX">
   <property name="text">
    <string>GPX...</string>
   </property>
  </action>
  <action name="actionImport_GeoJSON">
   <property name="text">
    <string>GeoJSON...</string>
   </property>
  </action>
  <action name="actionImport_CSV">
   <property name="text">
    <string>CSV...</string>
   </property>
  </action>
  <action name="actionEdit_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEdit_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionEdit_Cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/cut.svg</normaloff>:/icons/toolbar/cut.svg</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionEdit_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/copy.svg</normaloff>:/icons/toolbar/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionEdit_Paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/paste.svg</normaloff>:/icons/toolbar/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionEdit_Delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEdit_Properties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Properties...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionEdit_Rename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename...</string>
   </property>
  </action>
  <action name="actionFile_ConnectToDatabase">
   <property name="text">
    <string>Connect to Database...</string>
   </property>
  </action>
  <action name="actionView_Crosshairs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Crosshairs</string>
   </property>
  </action>
  <action name="actionDrawMarker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/marker-add.svg</normaloff>:/icons/toolbar/marker-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Marker</string>
   </property>
  </action>
  <action name="actionFile_SwitchWorkspace">
   <property name="text">
    <string>Switch Workspace...</string>
   </property>
  </action>
  <action name="actionFile_DisconnectDatabase">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSortByName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionSortByCustomOrder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom Order</string>
   </property>
  </action>
  <action name="actionObjectAddFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/toolbar/image-add.svg</normaloff>:/icons/toolbar/image-add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionObject_Browser">
   <property name="text">
    <string>Browser...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SlippyMapWidget</class>
   <extends>QWidget</extends>
   <header location="global">SlippyMap/SlippyMapWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlippyMapTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">Map/SlippyMapTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>OEDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">Widget/OEDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
